examples/webgpu_postprocessing_ssr.html

```diff
@@ -75,6 +75,12 @@
 
 					if ( object.material ) {
 
+						if ( object.material.name === 'Lense_Casing' ) {
```

Comment:
Maybe reverting this addition due to https://github.com/mrdoob/three.js/pull/31539#issuecomment-3146164930?

Yes, the previous look was also not physically correct but the missing objects when looking through the lens makes the camera look broken^^.